### PR TITLE
Israeli Supermarkets

### PR DESCRIPTION
**Pull request by yotamN**

### CODE REVIEW

File: data/brands/shop/cheese.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "items": [
+    {
+      "displayName": "באשר פרומז'רי",
+      "locationSet": {"include": ["il"]},
+      "tags": {
+        "brand": "Basher Fromagerieil",
+        "brand:wikidata": "Q122364246",
+        "name": "באשר פרומז'רי",
+        "name:he": "באשר פרומז'רי",
+        "name:en": "Basher Fromagerieil",
```
Comment (AntiCompositeNumber) [inline data/brands/shop/cheese.json:14]:
```suggestion
        "name:en": "Basher Fromagerie",
```

File: data/brands/shop/cheese.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "properties": {
+    "path": "brands/shop/cheese",
+  },
+  "items": [
+    {
+      "displayName": "באשר פרומז'רי",
+      "locationSet": {"include": ["il"]},
+      "tags": {
+        "brand": "Basher Fromagerieil",
```
Comment (AntiCompositeNumber) [inline data/brands/shop/cheese.json:10]:
```suggestion
        "brand": "באשר פרומז'רי",
```
Better to use the more common language for the brand.

File: data/brands/shop/supermarket.json [hunk 3]
```diff
@@ -9476,6 +9477,19 @@
         "shop": "supermarket"
       }
     },
+    {
+      "displayName": "טיב טעם בעיר",
+      "locationSet": {"include": ["il"]},
+      "tags": {
+        "alt_name": "Tiv Ta'am in The City",
+        "brand": "Tiv Ta'am in The City",
+        "brand:wikidata": "Q122189282",
```
Comment (AntiCompositeNumber) [inline data/brands/shop/supermarket.json:9486]: https://www.wikidata.org/wiki/Q122189282 was speedily deleted as not meeting [Wikidata notability](https://www.wikidata.org/wiki/Wikidata:Notability). Please see https://github.com/osmlab/name-suggestion-index/pull/8475#issuecomment-1665543134 and https://www.wikidata.org/wiki/Wikidata:Guide_to_requests_for_undeletion for more information.

File: data/brands/shop/supermarket.json [hunk 4]
```diff
@@ -9627,6 +9641,21 @@
...
+    {
+      "displayName": "יוחננוף",
+      "locationSet": {"include": ["il"]},
+      "tags": {
+        "brand": "יוחננוף",
+        "brand:en": "Yochananof",
+        "brand:he": "יוחננוף",
+        "brand:wikidata": "Q25492676",
+        "brand:wikipedia": "he:רשת יוחננוף",
+        "name": "יוחננוף",
```
Comment (AntiCompositeNumber) [inline data/brands/shop/supermarket.json:9653]:
```suggestion
        "brand:wikidata": "Q25492676",
        "name": "יוחננוף",
```
NSI no longer includes `brand:wikipedia`, see  #6481.

**Review by AntiCompositeNumber — CHANGES_REQUESTED** [review]

**Comment by AntiCompositeNumber** [comment]
@yotamN Your input on the suggestions would be appreciated. Otherwise, I intend to split Tiv Taam in The City out, apply the other suggestions, and merge.

**Comment by yotamNimble** [comment]
Sorry, will try to get to it tomorrow

**Comment by yotamN** [comment]
Sorry, I've been busy lately. Will try to find time today or in coming days to handle the Tiv Taam case although I'm not sure if I should insist of it being in Wikidata or not.

File: data/brands/shop/supermarket.json (hunk 3 above)
Comment (Flatroy) [reply data/brands/shop/supermarket.json:9539]: we already have this place with brand Tiv Ta'am / brand:wikidata: Q2903718

**Comment by yotamN** [comment]
I've fixed your comments

File: data/brands/shop/supermarket.json (hunk 3 above)
Comment (AntiCompositeNumber) [reply data/brands/shop/supermarket.json]: It's not ideal, but we can't control what Wikidata does.

**Review by AntiCompositeNumber — APPROVED** [review]

**Pull request merged**